flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/reader/deserializer/PulsarDeserializationSchema.java

```diff
@@ -76,7 +76,10 @@ default void open(InitializationContext context) throws Exception {
      * @param message The message decoded by pulsar.
      * @param out The collector to put the resulting messages.
      */
-    void deserialize(Message<byte[]> message, Collector<T> out) throws Exception;
+    void deserialize(Message<?> message, Collector<T> out) throws Exception;
```

Comment:
This is a `PublicEvolving` interface, change the method should add another method and add a `Deprecated` annotation for old method.

Reply:
When I create 2 methods at the same time, they clash with each other. Wondering is there a workaround ?